instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -118,7 +118,7 @@ def collect_request_attributes(environ):
     }
 
     host_port = environ.get("SERVER_PORT")
-    if host_port is not None:
+    if host_port is not None and not host_port == '':
```

Comment:
Could this be replaced by a check for `host_port`

`if host_port:`

Reply:
That will not work if host_port has a values like 0, 00, 000 etc